bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,22 +1,19 @@
 #KickBall
```

Comment:
Maybe we should change `#KickBall` to `#GoAndKickBall` or something similar because it now does more than the name `KickBall` suggests.

Reply:
I agree. In fact, I'm still not happy with `GoAndKickBall` because it does not reflect that the current implementation of `GoToBall` results in the robot standing behind the ball aligned towards the opponent's goal. Meaning `GoToBall` is currently only capable of direct kicks towards the goal and not of team play.